recipes/stringtie/1.2.4/meta.yaml

```diff
@@ -0,0 +1,25 @@
+about:
+  home: http://ccb.jhu.edu/software/stringtie/
+  license: Artistic License 2.0
+  summary: Transcriptome assembly and quantification for RNA-seq
+build:
+  number: 0
+  skip: True # [osx]
+package:
+  name: stringtie
+  version: 1.2.4
+requirements:
+  build:
+    - gcc   # [linux]
+    - llvm  # [osx]
```

Comment:
Well, it builds.

Let me see if I can build it with OSX. I copied over the meta file from 1.2.4, but lets see.